### PR TITLE
workflows: Allow configuring a separate executor pool for workflow actions

### PR DESCRIPTION
**Pull request by bduffany**
---

**Version bump**: Patch

### CODE REVIEW

File: enterprise/server/workflow/service/service.go [hunk 1]
```diff
@@ -419,6 +420,14 @@ func (ws *workflowService) createActionForWorkflow(ctx context.Context, wf *tabl
 	return actionDigest, err
 }
 
+func (ws *workflowService) workflowsPoolName() string {
+	cfg := ws.env.GetConfigurator().GetRemoteExecutionConfig()
+	if cfg != nil && cfg.WorkflowsPoolName != "" {
+		return cfg.WorkflowsPoolName
+	}
+	return "default"
```
Comment (bduffany) [inline enterprise/server/workflow/service/service.go:428]: refers to https://github.com/buildbuddy-io/buildbuddy/blob/62f1f2cf940a044cc12c3fa478d31129a2bbdcfd/enterprise/server/remote_execution/execution_server/execution_server.go#L440

Comment (tylerwilliams) [reply enterprise/server/workflow/service/service.go:428]: Could we make that "default" a const in https://github.com/buildbuddy-io/buildbuddy/blob/enterprise/server/remote_execution/platform/platform.go?

**Review by tylerwilliams — CHANGES_REQUESTED** [review]

File: enterprise/server/workflow/service/service.go (hunk 1 above)
Comment (bduffany) [reply enterprise/server/workflow/service/service.go]: Done, good call

**Review by tylerwilliams — APPROVED** [review]
LGTM

**Pull request merged**